sdk/storage/azure_storage_blob/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "azure_storage_blob"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```

Not necessary, but I'd add the following as part of your scaffolding since you know you'll need it, and it makes a good example:

```suggestion
[dependencies]
azure_storage_common = { workspace = true }
```

---

Good callout, took this feedback! Thanks for the review Heath I appreciate the help 😄